docs-parts/computation/04-master-part_lang3.rst

```diff
@@ -0,0 +1,22 @@
+
+.. code-block:: python
+
+  @schema
+  class ArrayResponse(dj.Computed):
+  definition = """
+  array_id: int
+  """
+
+    class ElectrodeResponse(dj.Part):
+    definition = """
+    -> master
+    electrode_id: int
```

Comment:
Let's rename `electrode_id` to `electrode`. optionally, add a comment explaining that this is the  electrode number on the probe.